misc/docker-ci/Dockerfile

```diff
@@ -8,15 +8,15 @@ RUN apt-get install --yes php-cgi
 
 # tools for building and testing
 RUN apt-get install --yes apache2-utils cmake cmake-data git memcached netcat-openbsd nghttp2-client redis-server wget sudo
-RUN apt-get install --yes libev-dev libc-ares-dev libnghttp2-dev libssl-dev libuv1-dev zlib1g-dev
+RUN apt-get install --yes libev-dev libc-ares-dev libnghttp2-dev libssl-dev libuv1-dev zlib1g-dev libbrotli-dev
 
 # clang-4.0 for fuzzing
 RUN apt-get install -y clang-4.0
 ENV PATH=/usr/lib/llvm-4.0/bin:$PATH
 
 # curl with http2 support
 RUN wget --no-verbose -O - https://curl.haxx.se/download/curl-7.57.0.tar.gz | tar xzf -
-RUN (cd curl-7.57.0 && ./configure --prefix=/usr/local --with-nghttp2 --disable-shared && make && sudo make install)
+RUN (cd curl-7.57.0 && ./configure --prefix=/usr/local --without-brotli --with-nghttp2 --disable-shared && make && sudo make install)
```

Comment:
Added `--without-brotli` because it doesn't compile. I guess there's a version mismatch in curl and brotli.